.github/workflows/test.yml

```diff
@@ -78,6 +78,15 @@ jobs:
       with:
         bats-version: 1.9.0
 
+    - name: procfs mount
+      run: |
+        # Get the list of mounts to help with debugging.
+        cat /proc/self/mounts
+        # Create a procfs mount that is not masked, to ensure that container
+        # procfs mounts will succeed.
+        sudo mkdir -p /tmp/.procfs-stashed-mount
+        sudo unshare -pf mount -t proc -o subset=pid proc /tmp/.procfs-stashed-mount
```

Comment:
I don't know why this has become necessary with this PR (the "problem" commit is the `/proc/self/exe` cloning change) but this solves the issue and this is a CI-weirdness issue.

Reply:
@cyphar Does that mean that runc won't work as-is inside "your usual" GHA, or is it only for the sake of testing?